### ISSUE TITLE
Translations from profiler don't map

### ISSUE THREAD

**Issue by Puddingboy** [bolt/bolt#7923]
Translations from the symfony translation panel don't map to the record. You can hit the edit button,  change the value and even pressing the save and sync button show nothing unusual, except the submitted value doesn't map to the record. Changes are saved in the panel, which are visible if you attempt to edit the fields again, but they either don't submit properly or they are not retrieved properly.

![new xpertdata nl__profiler_9946e7_panel=translation (1)](https://user-images.githubusercontent.com/4963787/125450916-5ddb5855-d2b0-4205-8a94-5f303c69f8cc.png)
This button doesnt do anything. It shows a loading icons for a couple of seconds and a message "Started synchronization of all translations", except nothing happens. Could also be stuck in a loop.

![new xpertdata nl__profiler_9946e7_panel=translation (3)](https://user-images.githubusercontent.com/4963787/125452271-ab022806-a55a-4ca6-9556-380365819f2c.png)
Fallback always shows 0.

![new xpertdata nl__profiler_9946e7_panel=translation](https://user-images.githubusercontent.com/4963787/125450948-7c38de2d-3e96-4e16-92b0-799809b586ea.png)
Neither the sync buttons, nor the add selected messages seems to do the trick. They show the same loading animation and also nothing further seems to happen.

![new xpertdata nl__profiler_9946e7_panel=translation (2)](https://user-images.githubusercontent.com/4963787/125451662-1154f8ce-5999-4e0e-8d27-96ea59e648b2.png)
This is what you see when you press the edit button.

I took the effort to browse through all languages of the localeswitcher on the frontend to make sure it wasn't accidentally mapped to the wrong language for whatever reason.

This is my boltforms field config for the record.

        fields:
            company:
                type: text
                options:
                    required: true
                    label: companyname
                    constraints: [ NotBlank, { Length: { 'min': 2, 'max': 128 } } ]
            name:
                type: text
                options:
                    required: true
                    label: name
                    constraints: [ NotBlank, { Length: { 'min': 2, 'max': 128 } } ]
            phonenumber:
                type: text
                options:
                    required: true
                    label: phonenumber
                    constraints: [ NotBlank, { Length: { 'min': 2, 'max': 128 } } ]
            email:
                type: email
                options:
                    required: true
                    label: email
                    constraints: [ Email ]
                    help: "test"
            message:
                type: textarea
                options:
                    required: true
                    label: message
                    attr: 
                        rows: 6
            contactpreference:
                type: choice
                options:
                    label: contactpreference
                    choices:
                        email: email
                        phonecall: phonecall
                    multiple: false
                    expanded: true
            submit:
                type: submit
                options:
                    label: submit
                    attr:
                         class: 'form-button'

This shows my services.yaml setup. no other values have been changed:
`parameters:
    locale: 'nl'
    # This parameter defines the codes of the locales (languages) enabled in the application
    app_locales: nl|en|fr|cz|sk
    app.notifications.email_sender: anonymous@example.com
    bolt.table_prefix: bolt_
    bolt.backend_url: /bolt

services:
    # default configuration for services in *this* file
    _defaults:
        autowire: true      # Automatically injects dependencies in your services.
        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
        public: false       # Allows optimizing the container by removing unused services; this also means
                            # fetching services directly from the container via $container->get() won't work.
                            # The best practice is to be explicit about your dependencies anyway.
        bind:               # defines the scalar arguments once and apply them to any service defined/created in this file
            $locales: '%app_locales%'
            $defaultLocale: '%locale%'
            $emailSender: '%app.notific`

This shows my contentypes config for the record:
`pages:
    name: Pages
    singular_name: Page
    fields:
        title:
            type: text
            localize: true
            class: large
            group: content
        teaser:
            type: textarea
            localize: true
            postfix: |
                A text-only version of the content, that can be used to "tease"
                the content when it's used in listing pages or search results.
        image:
            type: image
            localize: true
            label: Highlighted image
        body:
            type: redactor
            localize: true
            height: 300px
        slug:
            type: slug
            localize: true
            uses: title
            group: Meta
        laik:
            type: text
            group: Meta
        template:
            type: templateselect
            filter: '/^(?!partials)[^_].*\.twig$/'
    locales: ['nl', 'en', 'fr']
    records_per_page: 5
    order: id`

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 5.0.0
| Install type                   |  composer install bolt:setup
| BC Break                     | no
| PHP version                 | 7.3.28
| Web server                  | Apache
| For UX/UI issues        | Brave, chromium, windows10

Reproduction
------------

If you're filing a bug, please describe how to reproduce it. Include as much
relevant information as possible, such as:

### Bug summary

Translated labels don't save properly. The record cannot show the changes. The profiler translation backend shows the edited fields while still unrefreshed. It shows the edited value if you hit edit again. This either means they don't write properly to their locations or they are not retrieved properly.

### Specifics

Any yourdomain.TLD/_profiler/ID?panel=translation should show the same behavior.
the translations folder is currently only holding a .gitignore file
it fails to produce the xml type files which would normally store the values.

### Expected result

 * values saved, stored and visible on page

### Actual result

* loading and then no effect.

**Comment by Puddingboy** [comment]
This may also be relevant:
![__trans](https://user-images.githubusercontent.com/4963787/125751074-a520215f-1aa9-47da-a2a4-059ff1ac8180.png)

**Comment by bobdenotter** [comment]
I'm not sure i follow.. Looking at that screenshot, you're editing the translations for 'English', and it seems they're saved, right? 

This is completely separate from translations of content (which is done in the bolt backend) 

And also completely separat from translations of Boltforms (which aren't supported at all).

Could you be more precise in what you're trying to achieve?

**Comment by bobdenotter** [comment]
You mentioned: 

> values saved, stored and visible on page

But what are you doing to show them on the page? What does the relevant twig look like?

**Comment by Puddingboy** [comment]
You're right about content translation being different from keyword translations

the keywords mapped in the profiler are either from |trans or %trans% or from labels in forms.

I am not sure what I am supposed to expect from the symfony translation backend, since it's still a bit new to me. I would think any change in value here should be visible on the frontend on the corresponding language immediately after save and sync.

They are not though. They save to the resources/translations/messages but they never actually appear on the frontend.

**Comment by bobdenotter** [comment]
@Puddingboy If you do this: 

`{{ 'contactpreference'|trans }}`, what does it show? The translated value "Contact me by" ?

**Comment by Puddingboy** [comment]
@bobdenotter they will show up in the list. The value just never updates on the user side of the page. What happens is follows: I see the new labels/keywords show up. I can edit and store them, but they will never show on the frontend.

**Comment by bobdenotter** [comment]
So, let me ask more specifically.. If you put this in a template: 

`{{ 'contactpreference'|trans }}`

What does it show on the rendered web page on the frontend. Please make a screenshot.

**Comment by Puddingboy** [comment]
It will show 'contactpreference'.
![new xpertdata nl_page_contact (5)](https://user-images.githubusercontent.com/4963787/125807569-2f71967d-c2b0-44b8-bcba-24736a4beca7.png)

if I visit the _profiler/d8d6cc?panel=translation page by hitting the red translation button:
I will see the untranslated keyword in the list:
![new xpertdata nl__profiler_1b4730_panel=translation](https://user-images.githubusercontent.com/4963787/125807865-8a86ab0e-d028-428a-a341-99cd0b2c2d6f.png)
If click the edit button on this line I will see it swapped for this(which I saved previously):
![new xpertdata nl__profiler_d8d6cc_panel=translation](https://user-images.githubusercontent.com/4963787/125808008-635c80fc-2e2e-443a-9645-00434b0bf350.png)

neither save nor sync update the label on the frontend
![new xpertdata nl__profiler_d8d6cc_panel=translation (1)](https://user-images.githubusercontent.com/4963787/125808862-d2f90595-88df-4ed4-9203-0ddf24639508.png)

frontend will remain like this:
![new xpertdata nl_page_contact (5)](https://user-images.githubusercontent.com/4963787/125807569-2f71967d-c2b0-44b8-bcba-24736a4beca7.png)
